src/main/java/unrc/dose/UserProfileService.java

```diff
@@ -0,0 +1,42 @@
+package unrc.dose;
+
+import com.google.gson.Gson;
+
+/**
+* Class which calls user profile's methods and transforms they to json.
+*/
+public final class UserProfileService {
+  /**
+  *Invokes the method which creates a user profile and transforms to json.
+  **/
+  public static String userprofile(Integer userId, String displayName, String twitterId) {
```

Comment:
this method's name doesn't convey what the method does, you should change the name to something like createProfile